### DISCUSSION TITLE
Templates metadata outside YAML frontmatter

### DISCUSSION THREAD

**Discussion by jcusimano** [BryanJenksCommunity/FAQ#139] [Q&A] [answered]
Hi Bryan, 
First of all, thanks for sharing your obsidian workflow, it really helped me a lot.
I have a few questions regarding templates metadata.
Considering for example this screenshot from one book note of your vault:
<img width="801" alt="Captura de Pantalla 2022-02-28 a la(s) 13 05 42" src="https://user-images.githubusercontent.com/83508288/156016522-2318b603-9aab-4db5-8d16-0175a86e96bb.png">

1. What is the purpose of this metadata outside the front-matter section other than showing relevant information when the note is opened? Are they there to aid some kind of dataview queries?
2. Why is this metadata between bacticks like `Author:` and not Author:: (which is the documented data annotation Key:: Value syntax for Inline fields used by Dataview plugin)? Is it only a style thing or it has a practical purpose also? I've seen many using a simple Key: Value syntax for this, but never `Key:` Value.
3. Lastly, what's the meaning of 🧠 tag before the inbox 📥? I haven't seen it explained in your tag taxonomy page.

Thanks!

**Comment by tallguyjenks** [comment, marked as answer]
1. mostly for visual but can also be searched with regular expressions
2. because that's just how i formatted it and its not formatted for dataview
3. i have multiple vaults in my main vault 🧠️  is my main zettelkasten vault

**Reply by jcusimano** [reply]
Thanks!
